blog/templates/blog/admin/post_detail.html

```
@@ -1,62 +1,132 @@
-        >
-            목록으로
-        </a>
-        <form method="post" action="{% url 'blog:admin-post-delete' post.id %}" class="inline-form">
-            {% csrf_token %}
-            <button type="submit" class="btn">삭제</button>
-        </form>
+    <div class="preview-shell preview-desktop post-detail-shell">
+      <article class="dailyblog_post_preview">
+        <header class="preview-post-header">
+          {% if post.preview_image %}
+          <div class="preview-hero"
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="preview-hero"
            style="background-image: url('{{ post.preview_image.url }}'); background-color: transparent;"> ] on line 90.

(tag-pair)